prime-router/src/main/kotlin/azure/ConditionMapper.kt

```diff
@@ -0,0 +1,138 @@
+package gov.cdc.prime.router.azure
+
+import gov.cdc.prime.router.ActionLogDetail
+import gov.cdc.prime.router.Metadata
+import gov.cdc.prime.router.UnmappableConditionMessage
+import gov.cdc.prime.router.fhirengine.utils.getCodeSourcesMap
+import gov.cdc.prime.router.fhirengine.utils.getObservations
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.Coding
+import org.hl7.fhir.r4.model.Observation
+
+interface ConditionMapper {
+    /**
+     * Attempt to find diagnostic conditions for a given test [code]
+     * @return a list of diagnostic conditions identified (could be empty)
+     */
+    fun lookupCondition(code: String): List<Coding>
```

Comment:
Nit: I would consider dropping this?  Might encourage callers to use the bulk impl